commerce_coordinator/settings/base.py

```diff
@@ -349,9 +349,6 @@ def root(*path_fragments):
 ECOMMERCE_ADD_TO_BASKET_API_PATH = '/basket/add/'
 TITAN_URL = 'replace-me'
 
-# Enterprise URL
-ENTERPRISE_URL = 'SET-ME-PLEASE'
-
 # Timeout for enterprise client
 ENTERPRISE_CLIENT_TIMEOUT = os.environ.get('ENTERPRISE_CLIENT_TIMEOUT', 15)
 ENTERPRISE_USER_CACHE_TIMEOUT = 600  # Value is in seconds
```

Comment:
Can we remove the unused `SYSTEM_ENTERPRISE_LEARNER_ROLE` and `ENTERPRISE_USER_CACHE_TIMEOUT`